gm_dotnet_managed/GmodNET/GameConsoleWriter.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+public class GameConsoleWriter : TextWriter
+{
+    private class Tier0
+    {
+        [DllImport("tier0", CallingConvention = CallingConvention.Cdecl)]
+        public static extern void Msg([MarshalAs(UnmanagedType.LPStr)] string msg);
+    }
```

Comment:
We don't need a whole class for just one method. Make Msg just a private static method of `GameConsoleWriter` class.

Reply:
:( ok